pkg/asyncapi/error.go

```diff
@@ -0,0 +1,62 @@
+package asyncapi
```

Comment:
I would move errors to a sparate package `erros` and create it similar like https://github.com/kubernetes/apimachinery/blob/master/pkg/api/errors/errors.go or simpler https://github.com/kyma-project/kyma/blob/master/components/console-backend-service/internal/gqlerror/errors.go .